src/components/shared/RadioButton/index.tsx

```diff
@@ -42,18 +45,21 @@ const COLOR: {
   GRAY59: '#969696',
 };
 
-const InputRow = styled.TouchableOpacity<IInputRowProps>`
-  justify-content: ${({ isColumn }): string =>
-    isColumn ? 'center' : 'flex-start'};
-  flex-direction: ${({ isColumn }): string => (isColumn ? 'column' : 'row')};
+const SCInputRow = styled.TouchableOpacity<IInputRowProps>`
+  justify-content: ${({ isLabelColumn }): string =>
+    isLabelColumn ? 'center' : 'flex-start'};
+  flex-direction: ${({ isLabelColumn }): string =>
+    isLabelColumn ? 'column' : 'row'};
   align-items: center;
   padding-vertical: 8px;
```

Comment:
I don't know this! Could you explain this???🙂

Reply:
Oh it is doing nothing. I forgot to remove this line!!

Reply:
wow!!!! cool :)🤟

Reply:
I have to remove this after creating a new issue(https://github.com/dooboolab/dooboo-ui-native/issues/51#issuecomment-569477659) Thank you!

Reply:
Thank u :)!!

Reply:
Please do not use `padding-vertical` or `padding-horizontal` in `styled-components`. They don't actually work correctly since `styled-components` aim to provide pure css guideline and they are not in there (They only works in `react-native`). Using `padding` attr and do something like `padding: 8px 0`.